docs/components/modules/ROOT/pages/resteasy-component.adoc

```diff
@@ -0,0 +1,152 @@
+[[resteasy-component]]
+= Resteasy Component
+//THIS FILE IS COPIED: EDIT THE SOURCE FILE:
+:page-source: components/camel-resteasy/src/main/docs/resteasy-component.adoc
+:docTitle: Resteasy
+:artifactId: camel-resteasy
+:description: Defines the Resteasy Endpoint.
+:since: 3.3
+:supportLevel: Stable
+:component-header: Only consumer is supported
+
+*Since Camel {since}*
+
+*{component-header}*
+
```

Comment:
Somewhat the documentation description looks missing though

Reply:
This should be ok, it has been introduced from a @djencks PR